Shaloo/ar 4947 chunksize (#166)
arcana-network/storage: usage.md

1. Install Storage SDK
2. Import `StorageProvider` from the Storage SDK package in the dApp. Call `init` method of `StorageProvider` and specify the Web3 wallet `provider` and the `appAddress` as input parameters. **Note:** Get the provider via the Auth SDK or third-party supported wallet. You can copy the appAddress from the [Arcana Developer Dashboard](https://docs.beta.arcana.network/docs/config_dapp) after registering your dApp
3. Use `StorageProvider` to:

>    - `upload` and push file data into the Arcana Store. **Note:** Save file DID that is returned after the file upload operation is successful.

Reviewer: This is a good opportunity to fix this, “Save file DID” is not English.

```suggestion
   - `upload` and push file data into the Arcana Store. **Note:** The newly saved file's DID is returned once the file upload operation is successful.
```

let shareURL = await dAppStorageProvider.files.getPublicFileURL(did);
``` -->

### Upload Chunk Size

> Arcana Storage SDK uses chunked uploading mode. Each file is split into chunks of default size 10MB and chunks are uploaded one by one. These chunks are further processed on the server, encrypted, and split into multiple parts before being stored in the Arcana Store. Chunked uploading helps to overcome the server's default upload size limitation and there is no server-side setting change required to upload a very large file size. Also, it helps in decentralizing data storage.

Reviewer: > Uses chunked uploading mode

What does that even mean? I suggest: “uploads files in chunks”.

>chunks of default size 10MB

I recommend: “chunks (10 MiB by default)”.

> You can change the default chunk size while uploading a file. Make sure the new chunk size is a multiple of 16 otherwise there may be issues in downloading the file.

Reviewer: > there may be issues in downloading the file

Could be worded more precisely.

```suggestion
You can change the default chunk size while uploading a file. Make sure the new chunk size is a multiple of 16 (bytes) otherwise you may face unexpected issues if you try to download the file.
```
